packages/react-vapor/src/components/collapsible/CollapsibleConnected.tsx

```diff
@@ -22,15 +29,54 @@ const mapStateToProps = (state: IReactVaporState, ownProps: CollapsibleOwnProps)
     return {expanded: ownProps.expandedOnMount};
 };
 
-const mapDispatchToProps = (dispatch: IDispatch, ownProps: CollapsibleOwnProps): CollapsibleDispatchProps => ({
+const mapDispatchToProps = (dispatch: IDispatch, ownProps: CollapsibleOwnProps) => ({
     onMount: () => dispatch(addCollapsible(ownProps.id, !!ownProps.expandedOnMount)),
     onUnmount: () => dispatch(removeCollapsible(ownProps.id)),
     onToggleExpandedState: (currentExpandedState: boolean) =>
         dispatch(setCollapsibleExpanded(ownProps.id, !currentExpandedState)),
 });
 
-export const CollapsibleConnected: React.ComponentClass<CollapsibleProps> = connect(
+export const COLLAPSIBLE_TIME = 150;

```

Comment:
We could add this one the `Defaults`
```suggestion
export const COLLAPSIBLE_EFFECT_DURATION = 150;

```